modules/machineset-manually-scaling.adoc

```diff
@@ -59,3 +81,12 @@ spec:
 ====
 +
 You can scale the machine set up or down. It takes several minutes for the new machines to be available.
+
+.Verification
+
+* You can verify that the selected machine has been deleted:
```

Comment:
"selected" seems to be used outside of the approved scenarios in the ISG:

> Use to refer to a check box, radio button, menu item or other interface element that is marked or highlighted after selection.

Maybe: 

s/that the  selected machine has been deleted/You can verify that you deleted the machine you wanted: